_data/cryptocurrencies.yml

```
@@ -415,6 +415,15 @@ websites:
       - totp
     doc: https://github.com/2factorauth/twofactorauth/pull/2321#issuecomment-273971890
 
+  - name: STEX
+    url: https://stex.com
+    img: stex.png
+    tfa:
+      - totp
+      - email
+      - u2f
+    doc: https://help.stex.com/en/articles/3486981
```

```suggestion
    doc: https://help.stex.com/en/articles/3486981#individual-account-security-settings
```
Update URL to point to further down the page so that 2FA instructions are more accessible.